src/test/java/com/ericsson/ei/test/utils/TestConfigs.java

```diff
@@ -49,7 +49,8 @@ public static synchronized void init() throws Exception {
     private static synchronized void setUpMessageBus() throws Exception {
         LOGGER.debug("Debug:setting up message buss");
 
-        LOGGER.debug("before setting up message buss: amqpBroker: " + amqpBroker + ", conn: " + connection + ",cf:" + connectionFactory);
+        LOGGER.debug("before setting up message buss: amqpBroker: " + amqpBroker + ", conn: " + connection + ",cf:"
```

Comment:
Capital letter on the first B. buss --> bus. Also write out all abbreviations, we shouldn't use them in the log messages.

Reply:
done